media/css/firefox/welcome.scss

```
@@ -42,6 +42,12 @@ $image-path: '/media/protocol/img';
 //* -------------------------------------------------------------------------- */
 // Page hero
 
+.mzp-c-hero {
+    .welcome-page4 & {
+        background: radial-gradient(circle at 60% 100%, $color-yellow-50 5%, $color-orange-70 94%);
```

Gradients are well supported but it might still be a good idea to include a solid base color here:

```suggestion
        background: $color-orange-70 radial-gradient(circle at 60% 100%, $color-yellow-50 5%, $color-orange-70 94%);
```